backend/code/m-booking/src/main/java/com/mbooking/model/Reservation.java

```diff
@@ -37,11 +37,11 @@
 	@OneToMany(fetch = FetchType.LAZY, cascade=CascadeType.ALL)
 	private List<ReservationDetails> reservationDetails;
 	
-	@ManyToOne(fetch = FetchType.LAZY, cascade = CascadeType.PERSIST)
+	@ManyToOne(fetch = FetchType.LAZY, cascade = CascadeType.ALL)
```

Comment:
Keep in mind that you will delete a manifestation in case you delete a reservation...